packages/common/src/languages.ts

```diff
@@ -108,11 +109,12 @@ export const languageCodeToLanguage = Object.freeze({
 
 type LanguageCodeToLanguage = typeof languageCodeToLanguage;
 
-export type LanguageCode = keyof LanguageCodeToLanguage;
-
 // There doesn't seem to be a way to get the object keys that has a type which
 // is an array of a union of string literals rather than an array of strings.
 // https://www.charpeni.com/blog/properly-type-object-keys-and-object-entries
-export const languageCodes = Object.keys(
-	languageCodeToLanguage,
-) as unknown as LanguageCode[];
+export const languageCodes = Object.keys(languageCodeToLanguage) as [
+	keyof LanguageCodeToLanguage,
+];
+
+export const LanguageCode = z.enum(languageCodes);
```

Comment:
I think zod objects are camelCase cause they're just consts? although we're not following this convention everywhere
```suggestion
export const languageCode = z.enum(languageCodes);
```

Reply:
I don't think we're very consistent with this. I think that having camelCase for the zod object and TitleCase for the type probably makes sense as sometimes you need to specifically reference the zod object rather than the type. On the other hand, it's quite nice not having it all as title case and just being able to do : Type or Type.safeParse